### PR TITLE
Feature: HTML Mode

### PR DESCRIPTION
**Pull request by jleandroperez**
@diegoreymendez + @aerych mind taking a quick look?

I've simplified `EditorDemoController` a little bit: nuked setters that weren't being used anywhere.

Suggestions would be very welcome!

### CODE REVIEW

File: Example/WordPress-Aztec-iOS/EditorDemoController.swift [hunk 1]
```diff
@@ -5,20 +5,20 @@ import Aztec
 
 class EditorDemoController: UIViewController
 {
-    private var bottomConstraint: NSLayoutConstraint?
+    private var bottomConstraint: NSLayoutConstraint!
 
 
     private (set) lazy var editor: AztecVisualEditor = {
-        let e = AztecVisualEditor(textView: self.textView)
+        let e = AztecVisualEditor(textView: self.richTextView)
```
Comment (diegoreymendez) [inline Example/WordPress-Aztec-iOS/EditorDemoController.swift]: I know this wasn't introduced by your PR but I'd take the opportunity to remove the `e` variable, both for having a non standard name, and for not being really necessary.

**Comment by diegoreymendez** [comment]
Just 1 optional comment @jleandroperez and then :shipit: .  Nice work.

File: Example/WordPress-Aztec-iOS/EditorDemoController.swift (hunk 1 above)
Comment (aerych) [inline Example/WordPress-Aztec-iOS/EditorDemoController.swift]: +1 I think I did that just to inspect the variable before it was returned, but its not something that should have shipped

**Comment by aerych** [comment]
Seconded! :shipit:

**Comment by jleandroperez** [comment]
Thanks gentlemen!!!

**Pull request merged**